src/components/App.tsx

```diff
@@ -1452,13 +1465,17 @@ class App extends React.Component<AppProps, AppState> {
       this.files = { ...this.files, ...opts.files };
     }
 
+    const libraryMenuStatus = !(
+      this.deviceType.isMobile || this.deviceType.isNonMobileSmallerScreen
+    );
+
```

Comment:
if this isn't going to be reused elsewhere, let's just inline it in the object.

Also, let's expand it so that when you read it you don't have to mentally evaluate it yourself:

```
isLibraryOpen: !this.deviceType.isMobile && !this.deviceType.isNonMobileSmallerScreen
```

With that said, instead of having `isNonMobileSmallerScreen`, it's better to instead have a `canFitSidebar` state (or a better naming). Being explicit/specific is always better if possible.

Reply:
Actually isNonMobileSmallerScreen represents screen size which don't allow sidebar

Reply:
yeah, but right now `isNonMobileSmallerScreen` only relates to the sidebar. If we add a different UI, it'll likely need its own breakpoint. As such we should make the naming clear.

Reply:
I agree, `isNonMobileSmallerScreen` kinda looks generic but its not so better if we can convey thats its specific to sidebar.